org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/ZigBeeBindingConstants.java

```diff
@@ -140,10 +140,7 @@ public static String getISO8601StringForDate(Date date) {
     }
 
     /**
-     * Convert a map into a json encoded string
-     *
-     * @param object
```

Comment:
Wouldn't it be better to add the Javadoc here rather than delete the empty references (which I assume was the "severe" error?).

Reply:
To avoid misunderstandings: I don't consider any of these severe, I just fixed them to get rid of the warnings in Eclipse, which I like to have enabled :)

In this case (as in the case below) I found the parameters to be so clear from the context that I considered it to be safe to remove them. But I see your point as well, so I will push a commit shortly including the removed docs.